.github/workflows/release-stable.yaml

```diff
@@ -24,11 +27,22 @@ jobs:
         run: sudo npm install -g vsce@latest
 
       - name: Install dependencies
-        run: npm install
+        run: npm ci
 
       - name: Verify analytics events
         run: npm run ampli:verify -- -t ${{ secrets.ITERATIVELY_KEY }}
 
+      - name: Bump patch version
+        id: patched-tag
+        uses: mathieudutour/github-tag-action@v6.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          release_branches: main
+          default_bump: patch
+
+      - name: Update package.json version
+        run: jq '.version = "${{ steps.patched-tag.outputs.new_version }}"' package.json > tmp.package.json && mv tmp.package.json package.json
```

Comment:
marvellous, jq is indeed available in action runners ⭐  much more readable then `sed` 😄